src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreCommand.cs

```diff
@@ -35,7 +36,27 @@ public class RestoreCommand
 
         public Guid ParentId { get; }
 
-        public const string ProjectRestoreInformation = "ProjectRestoreInformation";
+        // names for ProjectRestoreInformation and intervals
+        private static readonly string ProjectRestoreInformation = nameof(ProjectRestoreInformation);
+        private static readonly string Errors = nameof(Errors);
+        private static readonly string ErrorMessages = nameof(ErrorMessages);
+        private static readonly string Warnings = nameof(Warnings);
+        private static readonly string WarningMessages = nameof(WarningMessages);
+        private static readonly string Success = nameof(Success);
+
+        // names for child events for ProjectRestoreInformation
+        private static readonly string ExecuteRestoreAsyncInformation = nameof(ExecuteRestoreAsyncInformation);
```

Comment:
Would `GenerateRestoreGraph` be better event name or something similar which is more meaningful...